cli/cmd/check.go

```diff
@@ -66,6 +69,7 @@ non-zero exit code.`,
 	cmd.PersistentFlags().DurationVar(&options.wait, "wait", options.wait, "Maximum allowed time for all tests to pass")
 	cmd.PersistentFlags().StringVarP(&options.namespace, "namespace", "n", options.namespace, "Namespace to use for --proxy checks (default: all namespaces)")
 	cmd.PersistentFlags().BoolVar(&options.cniEnabled, "linkerd-cni-enabled", options.cniEnabled, "When running pre-installation checks (--pre), assume the linkerd-cni plugin is already installed, and a NET_ADMIN check is not needed")
+	cmd.PersistentFlags().StringVarP(&options.namespace, "output", "o", options.output, "Output format. One of: basic, json  (default: basic)")
```

Comment:
`cmd.PersistentFlags().StringVarP(&options.output`

also you can omit `(default: basic)`, as it's already printed as a default via `newCheckOptions` and cobra.

Reply:
good catch, thanks